test/OpenTelemetry.Exporter.Jaeger.Tests/JaegerExporterOptionsTests.cs

```diff
@@ -29,6 +29,7 @@ public JaegerExporterOptionsTests()
         public void Dispose()
         {
             ClearEnvVars();
+            GC.SuppressFinalize(this);
```

Comment:
Just as a FYI for these test classes which are `IDisposable` what I usually do to resolve a lot of warnings is just declare them `sealed`. Doing that relaxes a lot of the rules for proper dispose pattern because much of that pattern is there to make sure disposal works correctly in inheritance chains. Sealing class prevents those chains, which aren't particularly useful for one-off test classes anyway 😄

Reply:
I've done that in the past, too. Sealed works for the one where the analyzer wants the protected virtual Dispose implementation but doesn't stop the one about GC finalizer suppression.

Reply:
Interesting! I just tried in VS in a solution where I have analyzers running...

```csharp
		public class TestClass : IDisposable
		{
			// Generates CA1816 + CA1063
			public void Dispose()
			{
			}
		}

		public sealed class TestClass2 : IDisposable
		{
			// No warnings
			public void Dispose()
			{
			}
		}
```

Maybe it is a VS Code thing?

Anyway not a big deal just mainly curious.